naima/tests/test_models.py

```diff
@@ -523,3 +524,32 @@ def test_tablemodel():
     _ = IC.flux(e / 10)
     PP = PionDecay(tm)
     _ = PP.flux(e / 10)
+
+
+def test_eblabsorptionmodel():
+    """
+    test EblAbsorptionModel
+    """
+    from ..models import EblAbsorptionModel, BrokenPowerLaw
+ 
+    lemin, lemax = -4, 2
+   
+    EBL_zero = EblAbsorptionModel(0., 'Dominguez')
+    EBL_moderate = EblAbsorptionModel(0.5, 'Dominguez')
+   
+    e = np.logspace(lemin, lemax, 50) * u.TeV
+    amplitude = 10**31. / u.eV
+    e_break = (10**1.) * u.TeV
+    alpha1 = 0.35
+    alpha2 = 1.5
+    BPL = BrokenPowerLaw(amplitude, 1. * u.TeV, e_break, alpha1, alpha2)
+     
+    from ..radiative import InverseCompton
+    IC = InverseCompton(BPL,
+                        seed_photon_fields=['CMB'],
+                        Eemin=10 * u.GeV)
```

Comment:
Here we don't need to test the absorption on a computed IC spectrum because we only need to test the absorption, `InverseCompton` and `BrokenPowerLaw` are tested elsewhere. Can you use `np.ones_like(e)` as your source spectrum?